### PR TITLE
Feature/oniguruma lang

### LINKED ISSUE #452: use oniguruma with enry in language function
Add the possibility to compile gitbase using oniguruma integration with enry, to speed up language function.

Getting the language from 1M files:
- using Go regular expressions: ~17 min
- using oniguruma: ~4 min

We need to think about how to release binaries in a proper way including this.

### PR DESCRIPTION
**Pull request by erizocosmico**
Closes #452

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ before_install:
   - docker pull pilosa/pilosa:v0.9.0
   - docker run -d --name pilosa -p 127.0.0.1:10101:10101 pilosa/pilosa:v0.9.0
   - docker ps -a
+  - sudo apt-get install -y libonig2
```
Comment (kuba--) [inline .travis.yml]: `- sudo apt-get update`
?

**Comment by kuba--** [comment]
for linux we need to set up `-L` (GCC_FLAGS) or run `ldconfig` to link libonigurama

File: .travis.yml (hunk 1 above)
Comment (erizocosmico) [reply .travis.yml]: no need afaik

**Comment by erizocosmico** [comment]
@kuba-- it was missing the `libonig-dev` 😅

**Pull request merged**